diff --git a/src/emc/usr_intf/axis/scripts/image-to-gcode.py b/src/emc/usr_intf/axis/scripts/image-to-gcode.py
index 1e96de2dc9..c863e75008 100644
--- a/src/emc/usr_intf/axis/scripts/image-to-gcode.py
+++ b/src/emc/usr_intf/axis/scripts/image-to-gcode.py
@@ -30,8 +30,8 @@
 except ImportError:
     import Image
 
-import numpy.core
-plus_inf = numpy.core.Inf
+import numpy
+plus_inf = numpy.inf
 
 from rs274.author import Gcode
 import rs274.options
@@ -754,7 +754,6 @@
     if len(sys.argv) > 1:
         im_name = sys.argv[1]
     else:
-        import tkinter
         import tkinter.filedialog as tkFileDialog
 
         im_name = tkFileDialog.askopenfilename(defaultextension=".png",
@@ -762,8 +761,6 @@
                 (_("Depth images"), ".gif .png .jpg"),
                 (_("All files"), "*")))
         if not im_name: raise SystemExit
-        tkinter._default_root.destroy()
-        tkinter._default_root = None
     im = Image.open(im_name)
     size = im.size
     im = im.convert("L") #grayscale
